gemrb/plugins/TLKImporter/TLKImporter.cpp

```diff
@@ -55,7 +55,7 @@ TLKImporter::TLKImporter(void)
 	for(int i=0;i<gtcount;i++) {
 		ieVariable key;
 
-		strnuprcpy(key, tm->GetRowName(i), sizeof(ieVariable)-1 );
+		strnuprcpy(key.begin(), tm->GetRowName(i), sizeof(ieVariable)-1 );
```

Comment:
Is there some reason why all these strnlwrcpy/strnuprcpy/strlcpy calls weren't converted to regular assignment + case change (where applicable)?

Reply:
they are later on; those functions don't even exist anymore. I was merely trying to limit changes for easier commit digestion.